boards/arm/nucleo_wb55rg/Kconfig.board

```diff
@@ -8,3 +8,4 @@
 config BOARD_NUCLEO_WB55RG
 	bool "Nucleo WB55RG Development Board"
 	depends on SOC_STM32WB55XX
+	select CLOCK_STM32_LSE
```

Comment:
Please move in board's Kconfig.defconfig

Reply:
Done. for the future: should i put default configs into Kconfig.board, or Kconfig.defconfig?
For some boards there is selections in Kconfig.board